src/main/java/appeng/client/gui/implementations/GuiNetworkStatus.java

```diff
@@ -430,14 +430,18 @@ private void drawItemInfo() {
                 16,
                 GuiColors.DefaultBlack.getColor());
         this.fontRendererObj.drawString(
-                GuiText.Green.getLocal() + " : "
+                GuiText.Green.getLocal().charAt(0) + " : "
```

Comment:
* First character of color name isn't clear *enough* as a label.
* In some locales first character can even cause confusion.

Please find a nice way to display all color names without such abbreviation. And apply such rule in all similar cases below.